docs/dev_building_from_source.rst

```diff
@@ -0,0 +1,47 @@
+
+==============================
+Building libCellML from Source
+==============================
+
+This document covers building libCellML from source.  It is assumed that you already have the codebase downloaded and ready for building.  The variable LIBCELLML_SRC shall be used to refer to the directory containing the LICENSE file for libCellML.
```

Comment:
Do you think the name of the project needs to match the camel case used in the code? The name of the project is libCellML, but that doesn't work so well in actual code (hence LibCellml (upper-) or libCellml (lower camel case)). If its better they match, then I'd change my vote in the camel case discussion ;-)